ciao-controller/openstack_image.go

```diff
@@ -65,11 +67,14 @@ func (is ImageService) CreateImage(req image.CreateImageRequest) (image.DefaultR
 		CreateTime: time.Now(),
 	}
 
+	glog.Infof("Creating Image: %v", id)
```

Comment:
I'd prefer to see this line at the start of the function, even if we don't know what the ID is.  Having it down here means that it might not always get output.